app/src/main/java/net/skyscanner/backpack/demo/stories/Calendar2Story.kt

```diff
@@ -114,22 +118,25 @@ class Calendar2Story : Story() {
       Type.WithLabels -> {
         calendar.setParams(
           CalendarParams(
+            now = now,
             range = range,
             selectionMode = CalendarParams.SelectionMode.Range,
             cellsInfo = range
               .toIterable()
               .associateWith {
-                val price = random.nextInt(maxPrice)
+                val price = it.dayOfMonth % maxPrice
```

Comment:
Will this always be 1 % 5?

Reply:
Depending it date, it'll be x % 5, when x in [1, 31]

Reply:
Yeah I ignored the fact this the range.